resources/js/Pages/Home.vue

```diff
@@ -1,9 +1,63 @@
 <template>
     <div class="page-container home-page">
         <Head title="Mismatch Finder" />
-        <section id="intro-section">
-            <h2 class="h4">{{ $i18n('about-mismatch-finder-title') }}</h2>
-            <p id="about-description" >{{ $i18n('about-mismatch-finder-description') }}</p>
+        <section id="description-section">
+            <header class="description-header">
+                <h2 class="h4">{{ $i18n('about-mismatch-finder-title') }}</h2>
+                <wikit-button
+                    class="instructions-button"
+                    variant="quiet"
+                    type="progressive"
+                    @click.native="$refs.faq.show()"
+                >
+                    <template #prefix>
+                        <icon type="info-outlined" size="medium" color="inherit"/>
+                    </template>
+                    {{ $i18n('faq-button') }}
+                </wikit-button>
+            </header>
+
+            <wikit-dialog class="instructions-dialog"
```

Comment:
same thing: this could be "faq-dialog" for more clarity

Reply:
+1